schema.graphql

```diff
@@ -815,6 +873,21 @@ input DeleteProjectInput {
   projectId: ID!
 }
 
+input LocationInput {
+  zone: String!
+  section: String!
+  area: String!
+}
+
+input WidgetLayoutInput {
+  extended: Boolean
+	Location: LocationInput!
```

Comment:
Not needed because there are no duplicated widget id so we can identify specific widget by one id